### PR TITLE
Merge pull request #10188 from Microsoft/users/aamallad/project_scope…

### PR DESCRIPTION
**Pull request by aasim**
…d_feeds

Added support for project scoped feeds

### CODE REVIEW

File: Tasks/NuGetRestoreV1/task.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 1,
-        "Minor": 148,
+        "Minor": 152,
```
Comment (jotaylo) [inline Tasks/NuGetRestoreV1/task.json]: 151

File: Tasks/TwineAuthenticateV0/task.json [hunk 2]
```diff
@@ -9,8 +9,8 @@
     "category": "Package",
     "version": {
         "Major": 0,
-        "Minor": 151,
-        "Patch": 1
+        "Minor": 152,
```
Comment (jotaylo) [inline Tasks/TwineAuthenticateV0/task.json]: 151
2

File: Tasks/NuGetRestoreV1/task.json (hunk 1 above)
Comment (aasim) [reply Tasks/NuGetRestoreV1/task.json]: Fixed

File: Tasks/TwineAuthenticateV0/task.json (hunk 2 above)
Comment (aasim) [reply Tasks/TwineAuthenticateV0/task.json]: Fixed

**Pull request merged**